timeline/templates/timeline/timeline.html

```diff
@@ -1,8 +1,9 @@
-{% extends 'projectroles/project_base.html' %}
```

Comment:
These edits to `timeline.html` are incorrect. As we agreed yesterday in Teams, this file should remain as is, as it is included in a project view and uses a different base template to site views.

The templates for the site-wide event view and admin view should be unified, as they use the same base template and should be 99% identical.

We should end up with two templates:

- `timeline.html` used for project view
    * Based on `project_base.html`, content under the `projectroles_extend` block
    * In other words, no changes to prior implementation
- `timeline_site.html`
    * Used for both site-wide event view and the admin view
    * Should only require slight edits to the current `timeline_site.html`
    * Most likely won't require much changes apart from than title/heading?
    * I noticed the existing template has erroneously some project access references included, please do remove these as we, by definition, never have a project context in a site app view

Reply:
For differences in project app and site app templates, see here:

- [Project app templates](https://sodar-core.readthedocs.io/en/dev/dev_project_app.html#template-structure)
- [Site app templates](https://sodar-core.readthedocs.io/en/dev/dev_site_app.html#templates)